content/en/blog/2024/contribute-as-sig-docs-reviewer.md

```diff
@@ -0,0 +1,111 @@
+---
+
+You've likely heard the adage, "Contributing isn't just about writing code",
+whispered in forums or seen etched into the digital walls of open source communities.
+But what depth of truth lies within this ancient wisdom, especially in the vast,
+evolving world of Kubernetes?
+
+Today, contributing to open source extends far beyond the realm of coding.
+It's an inclusive journey that welcomes the diverse skills of all who wish
+to be a part of it. Whether you're a seasoned developer or someone whose strengths
+lie outside traditional programming, your contributions hold immense value.
+
+Imagine yourself embarking on a journey that leverages your expertise and love
+for free software to enhance your capabilities and strengthen the very foundation
+of the Kubernetes ecosystem. This is a real opportunity for you to shine, to make
+an indelible mark on a project that powers some of the most critical infrastructure
+in the world today.
+
+Your journey begins with a single step: joining the Special Interest Group (SIG) Docs. 
+Here, your talent for articulating complex concepts, your keen eye for detail,
+and your unwavering commitment to clarity become your most powerful tools. As a contributor,
+you can transform dense technical landscapes into navigable pathways, making Kubernetes
+accessible to all.
+
+This is your call to adventure, an invitation to expand beyond the familiar territories
+of code and into the rich, uncharted domains of documentation and collaboration,
+empowering a global community of users and contributors.
+
+## Why become a SIG Docs reviewer?
+
+The Kubernetes project depends substantially on the SIG Docs to ensure that documentation
+is accurate, up to date, and easily accessible. By becoming a reviewer, you can help users
+and contributors navigate and understand Kubernetes more effectively. In addition, reviewing
+documentation provides unique opportunities for:
+
+- **Expanding your Kubernetes knowledge**: Engage deeply with new features and functionalities
+  by reviewing their documentation.
+- **Improving your technical writing skills**: Develop an eye for detail and clarity in technical
+  writing.
+- **Strengthening the Kubernetes community**: Help maintain the high quality of Kubernetes
+  documentation, supporting both new and experienced users.
+- **Building your network**: Expanding your professional network, and getting together with
+  contributors from around the globe.
+
+## Who are we looking for?
+
+We're seeking open source enthusiasts with:
+
+- Experience with Git and GitHub, comfortable with the process of reviewing pull requests
+  and providing constructive feedback.
+- Familiarity with Markdown and documentation frameworks (Hugo experience is a plus but
+  not required).
+- A passion for making complex technical concepts understandable and accessible.
+
+Prior experience in technical writing or documentation review in any open source project
+will be beneficial but not mandatory.
```

Comment:
```suggestion
Prior experience in technical writing or documentation review in any open source project
will be beneficial but not mandatory. We do need you to be very comfortable reading and writing
the language you'll be reviewing for (English is our main language).
```
You could mention what level of Kubernetes experience we look for: **not much**. If you have it, that's great;
if you don't yet know Kubernetes / containers, that means you bring a useful perspective that helps us check if documentation makes sense to a relative beginner.